MANIFEST.in

```diff
@@ -11,14 +11,13 @@ recursive-include pootle/locale *
 prune pootle/locale/*/.translation_index
 recursive-include pootle/translations *
 recursive-include pootle/settings *
-recursive-include pootle/static *
 recursive-include pootle/templates *
 recursive-include pootle/apps/*/templates *
-global-exclude tests.py
+recursive-exclude tests *
+recursive-exclude pootle/static *
```

Comment:
Note to self: Get back static assets so people installing from PyPI can recreate assets if they want to. Obviously exclude `node_modules`.